addons/binding/org.openhab.binding.pentair/src/main/java/org/openhab/binding/pentair/internal/PentairPacket_HeatSetPoint.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.pentair.internal;
+
+/**
+ * Pentair heat set point packet specialization of a PentairPacket. Includes public variables for many of the reverse
+ * engineered
+ * packet content.
+ *
+ * @author Jeff James - initial contribution
+ *
+ */
+public class PentairPacket_HeatSetPoint extends PentairPacket { // 29 byte packet format
+
+    final protected static int POOLTEMP = 5 + OFFSET;
```

Comment:
Non-preferred order of modifiers correct order is `public static final` See also: http://stackoverflow.com/questions/16731240/what-is-a-reasonable-order-of-java-modifiers-abstract-final-public-static-e#16732059\

Please check all